lib/susedistribution.pm

```diff
@@ -341,7 +341,8 @@ sub init_consoles {
                 {
                     hostname => $hostname,
                     password => $testapi::password,
-                    user     => 'root'
+                    user     => 'root',
+                    serial   => 'mkfifo /dev/sshserial; tail -f /dev/sshserial'
```

Comment:
where is this coming from? I doubt this will not have a negative impact on the backends which already use this console

Reply:
I will check with this changes